src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -186,13 +207,15 @@ def _run_create_mapping(app, args):
     try:
         deploy_cfg = get_deployment_config(app)
         log.info('Running create mapping on env: %s' % deploy_cfg['ENV_NAME'])
-        if args.wipe_es:  # override deploy_cfg WIPE_ES option
-            log.info('Overriding deploy_cfg and wiping ES')
-            deploy_cfg['WIPE_ES'] = True
-        run_create_mapping(app,
-                           check_first=(not deploy_cfg['WIPE_ES']),
-                           purge_queue=args.clear_queue,
-                           item_order=ITEM_INDEX_ORDER)
+        override_deploy_cfg(deploy_cfg, args.wipe_es, 'WIPE_ES')
+        override_deploy_cfg(deploy_cfg, args.skip, 'SKIP')
+        override_deploy_cfg(deploy_cfg, args.strict, 'STRICT')
```

Comment:
I think this should just be a dictionary
```
apply_dict_overrides(deploy_config, WIPE_ES=args.wipe_es, SKIP=args.skip, STRICT=args.strict)
```
where the following would be in misc_utils, as it would be generally useful in other contexts if it had semantics that were not specific to this app
```
def apply_dict_overrides(dictionary: dict, **overrides) -> dict:
    for k, v in overrides.items():
        if v is not None:
            dictionary[k] = v
    # This function works by side effect, but getting back the changed dict may be sometimes useful.
    return dictionary
```
In other words, I would suggest that if the dictionary entry contains None, it leaves the default and if it contains `True` or `False`, then you would override. This is maximally general for reuse, and yet you can still get the current semantics of your command by just changing the default in the `argparse` specs to be `None` instead of `False`.